### ISSUE TITLE
the "isInSameDayAsDate" does not seem to be available? 

### ISSUE THREAD

**Issue by callagga** [malcommac/SwiftDate#173]
I have a Swift project using SwiftDate however whilst I can see/use some of the SwiftDate NSDate extensions I am not seeing "isInSameDayAsDate" as available to use in XCode, even though I see this function in the doco/source code on github?

What do I need to do to get access to the "isInSameDayAsDate" in XCode?

My Podfile (using what was recommended on the user doco):
```
use_frameworks!
platform :ios, '8.0'
pod "SwiftDate", "~> 2.0"
```

PS.  Also wondering here was there a specific reason one has to put the v8 IOS platform statement in the Podfile, and also v2 only for SwiftDate?

**Comment by Hout** [comment]
You should try SwiftDate v3 and later.
Does that resolve your issue?

**Comment by callagga** [comment]
I get the following error message (below) if I drop the recommended Pod file lines of:

```
use_frameworks!
platform :ios, '8.0'
pod "SwiftDate", "~> 2.0"
```

Error message
```
$ pod install --verbose
  Preparing

Updating local specs repositories

Updating spec repo `master`
  $ /usr/bin/git pull --ff-only
  Already up-to-date.

CocoaPods 1.0.0.beta.4 is available.
To update use: `gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information see http://blog.cocoapods.org
and the CHANGELOG for this version http://git.io/BaH8pQ.


Analyzing dependencies

Inspecting targets to integrate
  Using `ARCHS` setting to build architectures of target `Pods`: (``)

Finding Podfile changes
  - SwiftDate

Resolving dependencies of `Podfile`
[!] Unable to satisfy the following requirements:

- `SwiftDate` required by `Podfile`
- `SwiftDate (= 2.0.2)` required by `Podfile.lock`

Specs satisfying the `SwiftDate` dependency were found, but they required a higher minimum deployment target.

Specs satisfying the `SwiftDate (= 2.0.2)` dependency were found, but they required a higher minimum deployment target.

/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/resolver.rb:411:in `handle_resolver_error'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/resolver.rb:69:in `rescue in resolve'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/resolver.rb:57:in `resolve'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/analyzer.rb:539:in `block in resolve_dependencies'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/analyzer.rb:537:in `resolve_dependencies'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/analyzer.rb:70:in `analyze'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:213:in `analyze'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:136:in `block in resolve_dependencies'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:135:in `resolve_dependencies'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:105:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:101:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/usr/local/bin/pod:22:in `load'
/usr/local/bin/pod:22:in `<main>'
$ 



```

**Comment by Hout** [comment]
Just try: `pod "SwiftDate"` without the `, "~> 2.0"`bit.
Does that work for you?

**Comment by Hout** [comment]
Fixed in 4994b9f
